app/src/main/java/de/qspool/clementineremote/backend/player/MyLibrary.java

```diff
@@ -61,6 +62,8 @@
 
     public final static int IDX_COUNT = 6;
 
+    public final static int IDX_YEAR = 6;
+
```

Comment:
Should the value of `IDX_YEAR` be a `7`? Or is it the same value as `IDX_COUNT` on purpose?